lib/js/src/rpc/enums/AppInterfaceUnregisteredReason.js

```diff
@@ -143,6 +143,14 @@ class AppInterfaceUnregisteredReason extends Enum {
         return AppInterfaceUnregisteredReason._MAP.UNSUPPORTED_HMI_RESOURCE;
     }
 
+    /**
+     * Get the enum value for RESOURCE_CONSTRAINT.
+     * @returns {String} - The enum value.
+     */
```

Comment:
Please align with other libs. Take text of RPC spec.

Reply:
@SilvioDC-Ford fixed in https://github.com/smartdevicelink/sdl_javascript_suite/pull/264/commits/07b2d177e818f95d584fb8f67e3638558f667bec